doc/admin-guide/files/records.config.en.rst

```diff
@@ -643,6 +644,8 @@ tr-out
 tr-pass
    Transparent pass through. This option is useful only for inbound transparent proxy ports. If the parsing of the expected HTTP header fails, then the transaction is switched to a blind tunnel instead of generating an error response to the client. It effectively enables :ts:cv:`proxy.config.http.use_client_target_addr` for the transaction as there is no other place to obtain the origin server address.
 
+no-throttle
```

Comment:
Hmmm, what's the use case here? if you don't want to throttle, wouldn't you just set proxy.config.net.connections_throttle arbitrarily high? Or, perhaps have an option (-1?) for proxy.config.net.connections_throttle to mean no throttle?

Or is the main idea here that you'd leave one port open for connections, such as a critical requests port? If so, we probably should document that use case, no?